model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -149,7 +149,7 @@ class DependencyGraphBuilder<D>(
      * the [DependencyGraph].
      */
     fun addDependency(scopeName: String, dependency: D): DependencyGraphBuilder<D> =
-        apply { addDependencyToGraph(scopeName, dependency, transitive = false) }
+        apply { addDependencyToGraph(scopeName, dependency, transitive = false, emptySet()) }
```

Comment:
Commit message nit: Missing dot at the end of the last sentence that starts with "Resolves".